src/annotator/anchoring/test/fake-pdf-viewer-application.js

```diff
@@ -129,6 +129,11 @@ class FakePDFViewer {
     this.viewer = this._container;
 
     this.eventBus = new EventEmitter();
+
+    /** @type {'auto'|'page-fit'|'page-width'} */
```

Comment:
I'm not sure. When I created our own definitions they did not have any shipped with the package itself. That might have changed since I last looked at it.